connect-examples/v2/php_payment/process-card.php

```diff
@@ -32,8 +32,9 @@
   $locations = $locations_api->listLocations();
   #We look for a location that can process payments
   $location = current(array_filter($locations->getLocations(), function($location) {
-    return !empty($location->getCapabilities()) &&
-      in_array('CREDIT_CARD_PROCESSING', $location->getCapabilities());
+    $capabilities = $location->getCapabilities();
+    return !empty($capabilities) &&
```

Comment:
return `is_array($capabilities) && in_array('CREDIT_CARD_PROCESSING', $capabilities);` is clearer